src/custom/components/SearchModal/Manage/ManageMod.tsx

```diff
@@ -0,0 +1,98 @@
+import { Trans } from '@lingui/macro'
+import React, { useState } from 'react'
+import { PaddedColumn, Separator } from '@src/components/SearchModal/styleds'
+import { RowBetween } from 'components/Row'
+import { ArrowLeft } from 'react-feather'
+import { Text } from 'rebass'
+import { CloseIcon } from 'theme'
+import styled from 'styled-components/macro'
+// import { Token } from '@uniswap/sdk-core'
+import { ManageLists } from 'components/SearchModal/ManageLists'
+import ManageTokens from 'components/SearchModal/ManageTokens'
+// import { TokenList } from '@uniswap/token-lists'
+import { CurrencyModalView } from '@src/components/SearchModal/CurrencySearchModal'
+import { ManageProps } from './index' // mod
+
+const Wrapper = styled.div`
+  width: 100%;
+  position: relative;
+  padding-bottom: 80px;
+`
+
+// const ToggleWrapper = styled(RowBetween)`
```

Comment:
For large blocks use block /* */ comments to reduce the noise when comparing changes please

Reply:
Got it (had to re-learn my `SHIFT + OPTION + A` vs `COMMAND + /` for mac :)